en/mulesoft/modules/ROOT/pages/ms_composer_invocable_flows.adoc

```diff
@@ -468,6 +468,45 @@ The example response is:
 }
 ----
 
+ifeval::["{product}"=="mulesoft"]
+== Publish the Flow to Salesforce or an Anypoint Organization
+
+After activating your flow, you can publish the flow to your Salesforce organization. You must have the xref:rpa-home::ms-automation-credits-usage-types.adoc[Automation Starter or Automation Advanced subscription package] to use this service. 
```

Comment:
I think these package names might need an Anypoint before them, but I am not 100% certain. I know the Integration packages do.